web/src/main/java/com/softserve/osbb/controller/EventController.java

```diff
@@ -39,10 +45,10 @@
 
     @RequestMapping(value = "/", method = RequestMethod.GET)
     public ResponseEntity<List<Resource<Event>>> findAllEvents() {
-        List<Event> eventList = eventService.getAllEvents();
         logger.info("Getting all events.");
+        List<Event> eventList = eventService.getAllEvents();        
```

Comment:
this looks wrong - your list could be null after this line of code.
suggestion here: create new list and use allAll method to fill it.